alerts/alerts.yml

```diff
@@ -0,0 +1,53 @@
+groups:
+  - name: Distribution bot. Unexpected exceptions.
+    rules:
+      - alert: DistributionBotUnexpectedErrors
+        expr: increase(distribution_bot_unexpected_exceptions{service="nor-reward-distribution-bot"}[1h]) > 0
+        labels:
+          severity: critical
+          app_team: tooling
+        annotations:
+          summary: "Distribution bot got an unexpected exception."
+          description: "Unexpected exception has happened."
+  - name: Distribution bot. Account balance.
+    rules:
+      - alert: DistributionBotLowAccountBalance
+        expr: distribution_bot_account_balance{service="nor-reward-distribution-bot"}/10^18 < 0.5
+        for: 15m
+        labels:
+          severity: warning
+          app_team: tooling
+        annotations:
+          summary: "Low account balance for distribution bot {{ $value }} ETH."
+  - name: Distribution bot. Distributions.
+    rules:
+      - alert: DistributionBotLowDistributions
+        expr: max_over_time(distribution_bot_rewards_distribution_status[15m]) != 2
+        for: 15m
+        labels:
+          severity: critical
+          app_team: tooling
+        annotations:
+          summary: "Distributor bot didn't distribute anything in 15 minutes."
+          description: "Distributor bot stuck distributing."
```

Comment:
Add `address` label of Staking Module to description

Reply:
Done